### PR TITLE
Add DC/OS universe for OpenWhisk and orchestration scripts

### PR DESCRIPTION
**Pull request by duynguyen**
The universe contains packages for:
* API Gateway
* CouchDB
* OpenWhisk controller
* OpenWhisk invoker

### CODE REVIEW

**Comment by duynguyen** [comment]
@ddragosd wdyt?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# OpenWhisk Deployment for Mesos
+
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+[![Build Status](https://travis-ci.org/apache/incubator-openwhisk-deploy-mesos.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-deploy-mesos)
+
+This repository is part of [Apache OpenWhisk](http://openwhisk.incubator.apache.org/) and can be used to deploy OpenWhisk to a Mesos cluster.
+
+## Subprojects
+
+* TODO
```
Comment (ddragosd) [inline README.md]: will we have subprojects in this repo ?

File: dcos-deploy-script/Makefile [hunk 2]
```diff
@@ -0,0 +1,127 @@
+DCOS_ENDPOINT ?= <DC/OS URL>
+APIGATEWAY_DNS ?= <gateway endpoint>
+OPENWHISK_UNIVERSE ?= <Universe URL>
+WSK_NAMESPACE ?= guest
+WSK_AUTH_TOKEN ?= <enter token here>
+
+cli:
```
Comment (ddragosd) [inline dcos-deploy-script/Makefile:7]: like with the universe docker bellow, WDYT about exposing a docker image with the dcos CLI and potentially wsk CLI that could be used to execute these scripts in any environments ?

**Review by ddragosd — APPROVED** [review]
Should we add a `Status` section in the README saying

> This module is under active development and is provided as-is for testing.

So that we can get feedback from multiple deployments before we consider it to be _production ready_ ?

File: dcos-deploy-script/Makefile (hunk 2 above)
Comment (duynguyen) [reply dcos-deploy-script/Makefile:7]: yep +1. Should it be in another PR though?

**Comment by duynguyen** [comment]
thanks @ddragosd , yes that makes sense. Added the status line.

**Comment by duynguyen** [comment]
@ddragosd , can we merge this in order to allow further PRs with parameterized constraints (from @tysonnorris ) and others to build upon it?

I will fix the openwhisk image version to prevent failed deployments by changed env vars in a new PR.

**Pull request merged**